ping-centre/PingCentre.jsm

```diff
@@ -201,8 +196,6 @@ class PingCentre {
     this._topic = options.topic;
     this._prefs = Services.prefs.getBranch("");
 
-    this._setPingEndpoint(options.topic, options.overrideEndpointPref);
```

Comment:
There is a comment above with "@param {string} options.overrideEndpointPref - optional pref for URL where the POST is sent."

We can probably delete that, right?

Reply:
Good point.